core/common/src/main/java/alluxio/RuntimeConstants.java

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
@aaudiber has more context on the bug (https://github.com/Alluxio/alluxio/pull/1905). basically, if code here moved to `Constants.java`, the static block of `Constants`and `Configuration` depends on each other, causing unstable build problem. Moving `RuntimeConstant` nested under `Constant` will not solve that problem